### PR TITLE
Updates for F2023

### PR DESCRIPTION
**Pull request by lambdathetadelta**
Includes updates for 

- SEO tags in <head>
- Interest Form link updates
- Removal of unnecessary code in css files

### CODE REVIEW

File: about.css [hunk 1]
```diff
@@ -1,4 +1,4 @@
-@import url("https://fonts.googleapis.com/css2?family=Raleway&swap=display");
+@import url("https://fonts.googleapis.com/css2?family=Raleway&display=swap");
```
Comment (lambdathetadelta) [inline about.css:1]: Correct query parameter is display=swap
https://css-tricks.com/google-fonts-and-font-display/

Comment (ericluuv) [inline about.css:1]: Correct query parameter is display=swap

https://css-tricks.com/google-fonts-and-font-display/

File: about.css [hunk 2]
```diff
@@ -83,15 +83,12 @@
     line-height: 1.7;
     margin-bottom: 0px;
     padding-bottom: 0px;
-    max-width: 90%;
```
Comment (ericluuv) [inline about.css:86]: widths not needed

File: about.html [hunk 3]
```diff
@@ -2,7 +2,7 @@
 <html lang="en">
 
 <head>
-    <title>Lambda Theta Delta | Fraternity | Irvine</title>
+    <title>About | Lambda Theta Delta</title>
```
Comment (ericluuv) [inline about.html:5]: Tailor every <title> tag to the specific page, improves SEO

File: imgs/background.webp
Comment (ericluuv) [inline imgs/background.webp:1]: Converted to webp as these files were large

File: index.html [hunk 4]
```diff
@@ -46,8 +46,6 @@
     </script>
     <script src="schema.js"></script>
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-
-    <!-- <meta name="google-site-verification" content=""/> -->
```
Comment (ericluuv) [inline index.html:50]: not needed anymore, google domains automatically verifies us as the owner of the domain

File: interest-form.css [hunk 5]
```diff
@@ -1,4 +1,4 @@
-@import url("https://fonts.googleapis.com/css2?family=Raleway:wght@400&family=Inter&swap=display");
```
Comment (ericluuv) [inline interest-form.css:1]: family=Inter already called in nav.css

File: program.css [hunk 6]
```diff
@@ -1,4 +1,4 @@
-@import url('https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@1,800&family=Raleway:wght@300&display=swap');
```
Comment (ericluuv) [inline program.css:1]: Open Sans wasn't used in program.html

File: skeleton.html [hunk 7]
```diff
@@ -2,25 +2,24 @@
 <html lang="en">
 
 <head>
-    <title>Lambda Theta Delta | Fraternity | Irvine</title>
```
Comment (ericluuv) [inline skeleton.html:5]: Updates to skeleton so future users will have to fill these out

File: about.css (hunk 1 above)
Comment (puruvian) [reply about.css:1]: Is this more efficient? Having trouble understanding the article.

File: about.html (hunk 3 above)
Comment (puruvian) [reply about.html:5]: Was this the only page that needed this fix?

File: skeleton.html (hunk 7 above)
Comment (puruvian) [reply skeleton.html:5]: I was wondering what skeleton was for...

**Comment by puruvian** [comment]
Seems overall ready to merge.

File: about.css (hunk 1 above)
Comment (ericluuv) [reply about.css:1]: Not more efficient, but the correct query parameter
Display is the key, swap is the value

Before it was just wrong lol, which was my bad

File: about.html (hunk 3 above)
Comment (ericluuv) [reply about.html:5]: No, this tittle should be made unique to each page
So the about page title would show case "About | Lambda Theta Delta"

File: skeleton.html (hunk 7 above)
Comment (ericluuv) [reply skeleton.html:5]: Skeleton so if we want to add a new page, we have a skeleton html layout ready to use

No need to copy paste a previous page delete a ton of data

File: about.html (hunk 3 above)
Comment (puruvian) [reply about.html:5]: Oh, I was asking if the other pages had been updated, it looks like they have been, gonna merge this in now.

**Pull request merged**